Google & Unity Partnership Announcement Post

MattMonasch opened the pull request:
**Please describe what your Pull Request does**

Adding a new post that announces both the partnership between Google and Unity and the new Unity version unveiled that contains new support for chromebooks.

Removed the featuring metadata from the Chrome OS Blueprint post and added featuring metadata to the Unity post.

Including new images for use in featuring and in the Unity post.

Commit:
🚧 (post) First draft of unity post

Unity is releasing a new build of their engine which includes ChromeOS support. This post describes
the changes and links out to Unity forums for more details.

Commit:
🚧 (post) Pretty-ifying Unity post

Committing prettier changes that I forgot to run when I committed the first draft.

Commit:
🚧 (post) Linting fixes

Renaming files to match linting rules and changing tags to match linting rules.

Commit:
🚀 (post) Updating Unity Post

Unity post has gone through a number of revisions and has a new hero graphic. It also has been
changed to be the featured post on the News & Stories page.

Commit:
🚀 (post) Fixing Unity Post Naming Conventions

Sass doesn't like &s in the url, and so they've been removed.

MattMonasch committed:
Merge branch 'trunk' into feature/unity-alpha-post

joycetoh8 commented on pages/en/posts/google-unity-partner-to-support-chromebooks.md:
```diff
@@ -0,0 +1,55 @@
+  - unity
+  - keyboard support
+  - mouse support
+  - large screen
```
```suggestion
  - large screens
```
nit, but I think the tag is actually "large screens" plural

MattMonasch replied:
Oh good catch!

kford55 reviewed (dismissed):
LGTM

joycetoh8 commented on pages/en/posts/google-unity-partner-to-support-chromebooks.md:
```diff
@@ -0,0 +1,55 @@
+
+Starting today with the alpha release of Unity 2021.2.0a14, any developer can begin building for Chrome OS. After selecting Android as the build target, developers can choose to also build for Chrome OS through the Player Settings. Allowing a Chrome OS artifact to be built from the same project and build target is convenient as it reduces duplicate code and streamlines the publishing pipeline. Ad ...
+
+#[Unity's Player Settings with x86 and x86_64 options](/images/posts/google-unity-partner-to-support-chromebooks/unity-x86-support.png)
```
```suggestion
![Unity's Player Settings with x86 and x86_64 options](/images/posts/google-unity-partner-to-support-chromebooks/unity-x86-support.png)
```

MattMonasch replied:
Huh! Weird that # did display the image. Going to change this locally and test since we don't have staging links currently and then I'll push changes back up. Thanks!

Commit:
🚀 (post) Fixing Tag and Image Ref In Unity Post

Had a few PR comments around a misspelling of a tag and using # instead of ! for displaying and
image.

joycetoh8 commented on pages/en/posts/google-unity-partner-to-support-chromebooks.md:
```diff
@@ -0,0 +1,55 @@
+date: 2021-04-21
+---
+
+Chrome OS users like to play Android games of all sorts, from card games to fully immersive open world RPGs. In the past year alone, time spent playing Android games on Chromebooks tripled. In order to bring the widest variety of games to our users, we enabled ARM games to run on Chrome OS [with Houdini](https://chromeos.dev/en/games/optimizing-games-publishing) and continue to invest heavily in  ...
```
```suggestion
Chrome OS users like to play Android games of all sorts, from card games to fully immersive open world RPGs. In the past year alone, time spent playing Android games on Chromebooks tripled. In order to bring the widest variety of games to our users, we enabled ARM games to run on Chrome OS [with Houdini](/{{locale.code}}/games/optimizing-games-publishing) and continue to invest heavily in Android gaming performance improvements. Our ultimate goal is to ensure that developers have the tools they need to build great games and empower them to bring incredible experiences to the platform.
```

Commit:
🚀 (post) Use Locale Code For Internal Links

Fixing a link inside the Unity post so that it respects the locale code a user has selected.
